src/modm/driver/pwm/tlc594x.hpp

```diff
@@ -76,86 +74,106 @@ class TLC594X
 	 * @param dots		initialize dot correction buffer with value, disable with -1
 	 * @param writeCH	write channels value to chip
 	 * @param writeDC	write dots value to chip
+	 * @param enable set blank pin low after initialization
 	 */
-	static void
-	initialize(uint16_t channels=0, uint8_t dots=63, bool writeCH=true, bool writeDC=true);
+	TLC594X(uint16_t channels=0, uint8_t dots=63, bool writeCH=true, bool writeDC=true, bool enable=true);
 
 	/// set the 12bit value of a channel
-	/// call transfer() to update the chip
-	static void
+	/// call writeChannels() to update the chip
+	void
 	setChannel(uint16_t channel, uint16_t value);
 
 	/// @param value	the 12bit value of all channels
-	/// @param update	write data to chip
-	static void
-	setAllChannels(uint16_t value, bool update=false);
+	void
+	setAllChannels(uint16_t value);
 
 	/// get the stored 12bit value of a channel
 	/// this does reflect the actual value in the chip
-	static uint16_t
+	uint16_t
 	getChannel(uint16_t channel);
 
-	/// set the 6bit dot correction value of a channel
-	/// call transfer() to update the chip
-	static void
+	/// set the 6bit dot correction valuewriteChannels of a channel
+	/// call writeChannels() to update the chip
+	void
 	setDotCorrection(uint16_t channel, uint8_t value);
 
 	/// @param value the 6bit dot correction value of all channels
 	/// @param update write data to chip
-	static void
-	setAllDotCorrection(uint8_t value, bool update=false);
+	void
+	setAllDotCorrection(uint8_t value);
 
 	/// get the stored 6bit dot correction value of a channel
 	/// this does reflect the actual value in the chip
-	static uint8_t
+	uint8_t
 	getDotCorrection(uint16_t channel);
 
 	/// transfer channel data to driver chip
-	static void
+	modm::ResumableResult<void>
 	writeChannels(bool flush=true);
 
 	/// transfer dot correction data to driver chip
-	static void
-	writeDotCorrection();
+	modm::ResumableResult<void>
+	writeDotCorrection(bool flush=true);
 
-	/// writes data from the input shift register to either GS  or DC register.
-	inline static void
+	/// writes data from the input shift register to either GS or DC register.
+	void
 	latch();
 
+	/// sets the blank pin low
+	modm_always_inline
```

Comment:
Let the compiler decide this, the implementation of `Xblank::set()` is not guaranteed to be inlineable.
Please remove all the `modm_always_inline` from the class.

Reply:
Removed all the modm_always_inline from the class.